yt/docker/ytsaurus/Dockerfile

```diff
@@ -4,39 +4,75 @@
 ARG BASE_REPOSITORY="ghcr.io/ytsaurus/ytsaurus-nightly"
 ARG BASE_IMAGE="latest"
 
+# Args for odin.
+ARG PYTHON_RUNTIME_VERSION="3.12"
```

Comment:
It looks inconsistent with python3.11 below

Reply:
I had some issues with python3.11, something was incompatible, I will try again and mention them here.

Reply:
Ok, I tried 3.11 and it seems to build fine now. Will try it on a real installation later, but for now let's use 3.11.